en/community/weblogs/index.md

```diff
@@ -47,6 +49,7 @@ out there, be sure to share!
 [rubyland]: http://rubyland.news/
 [ruby-weekly]: https://rubyweekly.com/
 [dev-ruby-tag]: https://dev.to/t/ruby
-[riding-rails]: http://weblog.rubyonrails.org/
+[riding-rails]: https://rubyonrails.org/blog/
```

Comment:
Maybe "Ruby on Rails Blog" instead of riding-rails? (To update used name above)

Reply:
Good call, update 👍